src/main/java/step3/util/StringUtil.java

```diff
@@ -0,0 +1,13 @@
+package step3.util;
+
+public final class StringUtil {
+    private static final String BASE_DELIMITER = "\\s*,\\s*";
+
+    public StringUtil() {
+
+    }
```

Comment:
접근 제어자가 잘못 들어간 것 같아요
Reflection 을 이용한 생성도 막아주기 위해 다음과 같은 방식도 있는데 관련해서 고민해보시면 좋을 것 같아요!!
```suggestion
    private StringUtil() {
        throw new AssertionError(String.format("%s class can not be instantiated", getClass().getName()));
    }
```

Reply:
앗 넵 감사합니다.